### ISSUE TITLE
One question

### ISSUE THREAD

**Issue by BX-xb** [PythonOT/POT#212]
Hi, I want to ask a question, how to use Wasserstain distance or the optimal transport distance to measure the distribution of two datasets with labels.  When I use semi-supervised setting, like 
model=ot.da.SinkhornTransport()
model.fit(Xs=Train_feature,ys=Class_training,Xt=Test_feature,yt=Class_testing)
I want to know the  Wasserstain distance between the Train feature and the test feature, how to get it?
Much appreciate!

**Comment by ncourty** [comment]
Hello @BX-xb , if you want to compare joint distributions of (features, labels), we might want to come up with a specific cost that combines both feature and label distances. You can take a look at this paper https://arxiv.org/abs/1705.08848 where we propose a way to do so in a context of domain adaptation.

**Comment by BX-xb** [comment]
Thank you for your answer. I try to use JDOT like this:
Y,Yb=classif.get_label_matrix(Class_training)
clf_jdot,dic= jdot.jdot_svm(Train_feature,Y,Test_feature,Class_testing,gamma_g=0.1,numIterBCD = 10, alpha=.1, lambd=1e1,ktype='linear')
but then some errors occur:
UserWarning: numItermax reached before optimality. Try to increase numItermax.
   result_code_string = check_result(result_code)

**Comment by ncourty** [comment]
Ok I am not sure about what you want to achieve exactly. If you need to use Wasserstein as a distance between joint distributions, I am just advocating the use of a cost function which couples a distance in term of features plus a distance in the label space (loss function). Then you can solve a regular OT problem with this cost (as in JDOT).
The error you get tells you that the problem you are trying to solve needs more iteration for the OT solver in the inner loop. Try to increase this value.
